modules/schema/schema/core/primitive_base.schema.json

```diff
@@ -19,6 +19,9 @@
       },
       "minItems": 1,
       "additionalItems": false
+    },
+    "pose" {
```

Comment:
I am not familiar with the schema, but it seems to me this should be per-stream, not per-primitive.

Again, since this is a not an actual pose, but a reference, maybe call it something other than just `pose`?

Reply:
My comment here regarding per-stream assumes the pose field references a separate pose (or "coordinate system") stream. 

Having different primitives in the same stream using different poses seems overly involved, what would the use case be...

Reply:
You are 100% correct, and this was wrinkle I don't like and pointed out in the spec some ugly "we would assume poses are the same for a stream".

The short version is that *stream metadata* has been a once time only concept up front.  That has turned out to be insufficient, especially in live data connectsion where you don't know everything ahead of time.

So this addition to the primitive_base will be removed, and we will add the ability to send stream metadata at anytime.

And I agree on the name.  calling it *pose_stream* is likely better.